demos/browser/app/meetingV2/meetingV2.html

```diff
@@ -367,6 +369,8 @@ <h1 class="h3 mb-3 font-weight-normal text-center">Select devices</h1>
             <option value="us-east-1">United States (N. Virginia)</option>
             <option value="us-east-2">United States (Ohio)</option>
             <option value="us-west-2">United States (Oregon)</option>
+            <option value="us-gov-west-1">GovCloud (US-West)</option>
+            <option value="us-gov-east-1">GovCloud (US-East)</option>
```

Comment:
This list only contains regions where Transcribe is available, so `us-gov-east-1` should not be available as an option. Meetings hosted in `us-gov-east-1` would need to use `us-gov-west-1` or use the `auto` option to start transcription.

Reply:
updated.